<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-ERPFO11" ignored="1" name="CHAutomationTest" tests="2" failures="1" timestamp="08 Apr 2020 19:18:33 GMT" time="12.537" errors="0">
  <testcase name="checkSuccessPage" time="0.016" classname="com.CH.qa.test.SuccessTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.CH.qa.test.SuccessTest.checkSuccessPage(SuccessTest.java:37)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- checkSuccessPage -->
  <testcase name="getTitle" time="0.016" classname="com.CH.qa.test.TellUsYourSelfTest"/>
  <testcase name="validateErrorMessage" time="0.0" classname="com.CH.qa.test.TellUsYourSelfTest">
    <ignored/>
  </testcase> <!-- validateErrorMessage -->
</testsuite> <!-- CHAutomationTest -->
